classes/patreon_options.php

```diff
@@ -93,7 +75,7 @@ function patreon_plugin_setup_page(){
         <div class="wrap">
 
             <div id="icon-options-general" class="icon32"></div>
-            <h1>Patreon Wordpress Settings</h1>
+			<h1>Patreon Wordpress Settings</h1>
```

Comment:
I think the old position is correct, this isn't a child of the divs, it's a sibling

Reply:
Thats definitely true. However thankfully it isnt present in the latest version of the file.